src/ResourceManager/Compute/Commands.Compute.Test/ScenarioTests/SqlIaaSExtensionTests.ps1

```diff
@@ -10,6 +10,115 @@
 	8) Test with incorrect Name and crrect Version
 	9) Test with incorrect Name and incorrect Version
 #>
+
+function Test-SetAzureRmVMSqlServerAKVExtension
+{
+	Set-StrictMode -Version latest; $ErrorActionPreference = 'Stop'
+
+	# Setup
```

Comment:
please replace all TAB symbols in this file to 4 Space symbols. That way this file will be properly formatted on github.